docs/sources/architecture/thing-definition.md

```diff
@@ -154,4 +154,49 @@ The channel group type is defined on the same level as the thing types and chann
 </channel-group-type> 
 ```
 
-When a thing will be created for a thing type with channel groups, the channel UID will contain the group ID in the last segment divided by a hash (#). If an Item should be linked to channel within a group, the channel UID would be `binding:multiChannelSwitchActor:myDevice:switchActor1#switch` for the XML example before.
\ No newline at end of file
+When a thing will be created for a thing type with channel groups, the channel UID will contain the group ID in the last segment divided by a hash (#). If an Item should be linked to channel within a group, the channel UID would be `binding:multiChannelSwitchActor:myDevice:switchActor1#switch` for the XML example before.
+
+## Properties
+The configuration descriptions explained in full detail [here](../architecture/configuration.md) are used in order to provide a flexible configuration mechanism for bindings. As a binding developer you will often have the demand to add meta data to your thing type that represent no configuration parameter. For this purpose the thing type definition provides the possibility to specify properties, e.g. the vendor or the model ID of the concrete thing:
```

Comment:
> The configuration descriptions explained in full detail

Why do you refer to this? The properties are imho now completely unrelated to the config descriptions.

> As a binding developer you will often have the demand to add meta data to your thing type

This is too fuzzy for me. I think we need to clearly say why they exist and how they are used. I don't have any good formulation at hand either, but that only proves that I am still struggling with having a clear picture of this new concept.